frontend/src/app/modules/header/header-item/header-item.component.html

```diff
@@ -28,7 +28,7 @@
         <mat-menu #menuTeam="matMenu">
             <button mat-menu-item routerLink="#" *ngFor="let team of teams" (click)="changeTeam(team.id)">
                 <i class="fa-solid fa-people-group"></i>
```

Comment:
```suggestion
        <mat-menu #menuTeam="matMenu" xPosition="before">
            <div class="menu-team-content">
                <button mat-menu-item routerLink="#" *ngFor="let team of teams" (click)="changeTeam(team.id)">
                    <i class="fa-solid fa-people-group"></i>
```
`xPosition="before"` I talked about in another comment. 

Also, would be great to have left and right padding in mat menu, because now blue _New Team_ button touches the border. It's easy to do and good to have) 
I would probably add something like `<div class="menu-team-content">` into `mat-menu` and give it `padding: 0 2px;`, but it's up to you.

Reply:
changes applied